crates/t-digest-lib/Cargo.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "tdigest-lib"
+version = "0.0.0"
```

You should specify the Rust edition so it doesn't default to `2015`.

```suggestion
version = "0.0.0"
edition = "2021"
```